web/src/pages/TracesPage/TraceTable.tsx

```diff
@@ -47,5 +48,5 @@ export function TraceTable({ runQuery }: TraceTableProps) {
 }
 
 export function traceDetailLink({ traceId }: { traceId: string }) {
```

Comment:
we might not need this function as is just wrapping `linkToTraceDetailPage`, should we adjust the signature?

Reply:
The signatures of `traceLink` of `<PersesTraceTable>` and `onClick` of `<ScatterChart>` are different, that's why I thought it's a good idea to create tiny wrapper functions to align them with an internal function, which only takes one required argument, the `traceId`.

But I'm not sure if `LinkToTraceDetailPage` should be in `TraceDetailPage.tsx` or it'd be better to put it in `TracingUI.tsx` or in a new file like `links.ts` or something.

Reply:
maybe a links file would be better I see this changing in the future to consider dev console.

Reply:
done